### ISSUE TITLE
WaziCloud Dashboard Graph Legend Labels are too long, can not be edited and are outside the graph display area

### ISSUE THREAD

**Issue by SvenZeisberg** [Waziup/WaziCloud#239]
WaziCloud Dashboard Graph Legend Labels are too long, can not be edited, and are outside the graph display area and therefore a lot of space on the display can not be used (see picture).
![wazicloud_display_graph_legend_bad](https://user-images.githubusercontent.com/62630067/108229218-e68fe380-713f-11eb-9a97-5b57c712a90f.gif)

**Comment by cdupont** [comment]
Good point, do you suggest to place the legend above? below? overlapping?
